### DISCUSSION TITLE
New Customer Milestone Is Out

### DISCUSSION THREAD

**Discussion by MucahitErdoganUnlu** [bounswe/bounswe2024group11#196] [General]
The second customer milestone is on the moodle. I think it will be better to have a seperate meeting for discussing the new milestone before the weekly meeting.

**Comment by Meminseeker** [comment]
[Here](https://moodle.bogazici.edu.tr/pluginfile.php/1625481/mod_assign/introattachment/0/Customer%20Milestone%202%20-%20Group%20Deliverable.pdf?forcedownload=1) is the link for the new assignment

**Comment by Meminseeker** [comment]
I suggest meeting on the following Monday, the 22nd of April.
